src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,40 @@
 
 class Animal {
   // write your code here
+
+  constructor(name, health) {
```

Comment:
set as default value
```suggestion
  constructor(name, health = 100) {
```